examples/oak/README.md

```diff
@@ -0,0 +1,28 @@
+# Amiga Brain Camera example
+
+1. Install the dependencies
+
```

Comment:
I'd recommend adding instructions for a venv here so people aren't installing these directly on their system

Reply:
Sure